packages/nimble-components/src/table/components/row/styles.ts

```diff
@@ -52,4 +85,32 @@ export const styles = css`
     :host([selected]) nimble-table-cell {
         --ni-private-table-cell-action-menu-display: block;
     }
-`;
+`.withBehaviors(
+    themeBehavior(
+        Theme.color,
+        css`
+            :host([selectable]:not([selected]):hover)::before {
+                background-color: ${hexToRgbaCssColor(White, 0.05)};
+            }
+
+            :host([selected])::before {
+                background-color: ${hexToRgbaCssColor(White, 0.2)};
+            }
+
+            :host([selected]:hover)::before,
+                background-color: ${hexToRgbaCssColor(White, 0.15)};
+    `
+    ),
+    themeBehavior(
+        Theme.dark,
+        css`
+            :host([selected])::before {
+                background-color: ${hexToRgbaCssColor(PowerGreen, 0.2)};
```

Comment:
I thought the working decision was to use `fillSelectedColor` and `fillHoverSelectedColor` here for consistency with other components.